functorch/test/test_pythonkey.py

```diff
@@ -643,6 +645,7 @@ def forward(self, x, y):
         assert torch.allclose(inputs[0].grad, cloned_inputs[0].grad)
         assert torch.allclose(inputs[1].grad, cloned_inputs[1].grad)
 
+    @patch.object(functorch_config, "use_functionalize", True)
```

Comment:
https://github.com/pytorch/pytorch/pull/84435

I think after this lands this shouldn't be needed.

Reply:
Fixed.